mocha-test-results.txt

```diff
@@ -0,0 +1,574 @@
+warn: DISABLE_INTERNAL_HTTPS is set.
```

Comment:
Is this results file checked in intentionally?

Reply:
no. doh!